src/main/java/org/meteordev/meteorbot/command/Commands.java

```diff
@@ -43,7 +45,14 @@ public void onReady(@NotNull ReadyEvent event) {
             commandData.add(command.build(new CommandDataImpl(command.name, command.description)));
         }
 
-        MeteorBot.BOT.updateCommands().addCommands(commandData).complete();
+        MeteorBot.BOT.updateCommands()
+            .addCommands(commandData)
+            .addCommands(
+                Commands.context(Command.Type.USER, "Tell them to read FAQ"),
+                Commands.context(Command.Type.USER, "Send installation guide"),
+                Commands.context(Command.Type.USER, "No old versions")
```

Comment:
I'd make the names constants and optionally move them to the respective command implementations for more maintainability

Reply:
hm, ok

Reply:
wait, but whrere would i move them

Reply:
ill just make them constants